lib/app/components/video_preview/video_preview.dart

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/utils/date.dart';
+import 'package:ion/generated/assets.gen.dart';
+import 'package:video_player/video_player.dart';
+import 'package:visibility_detector/visibility_detector.dart';
+
+class VideoPreview extends HookWidget {
```

Comment:
Maybe let's move `VideoPreview` inside of the chat feature because we have a different UI for the video preview in the other places

Reply:
the flow related to the stories

Reply:
discussed